### DISCUSSION TITLE
Credit cards

### DISCUSSION THREAD

**Discussion by ellielo** [NAIST-SE/AttractingContributors#322] [General]
How many credit cards do you have?
Which one is the best?

**Comment by Ozankey** [comment]
I have 4 credit cards with different credit limits. I wish I could close them all, but still need those for living. I like my CSR the best. It has 100k points, primary rental insurance, National Exc status, free Gloabal.

**Comment by hernessteve** [comment]
I have only two credit cards. The best for me is that accountnow phone number is always reachable on this [website](https://accountnow.pissedconsumer.com/customer-service.html).  In the past i used to have 5 different credit cards and honestly it didn't give me any privileges, only lots of confusion.
